lib/main.dart

```diff
@@ -7,10 +7,22 @@ import 'package:binsight_ai/screens/splash/wifi_page.dart';
 import 'package:binsight_ai/screens/connection/connect_page.dart';
 import 'package:binsight_ai/database/connection.dart';
 import 'package:go_router/go_router.dart';
+import 'package:sqflite/sqflite.dart';
+import 'package:binsight_ai/pub_sub/subscriber.dart';
+import 'package:web_socket_channel/io.dart';
+import 'dart:convert';
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
-  await getDatabaseConnection();
+  final Database database = await getDatabaseConnection();
+  final channel = IOWebSocketChannel.connect('http://54.214.80.15/subscribe');
```

Comment:
Location will actually be here
```suggestion
  final channel = IOWebSocketChannel.connect('http://54.214.80.15/api/model/subscribe');
```